cobigen/cobigen-core-parent/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/reader/ContextConfigurationReader.java

```diff
@@ -43,6 +43,10 @@
     /** XML Node 'context' of the context.xml */
     private ContextConfiguration contextNode;
 
+    private Path contextFile;
+
+    private String filePath;
```

Comment:
1. never used?
2. Never start again with paths stored in a string. Just use paths. In addition, this variable is redundant to contextFile.

Reply:
1. They are used in the readConfiguration() method
2. I just adapted this as it was already implemented